release-please-config.json

```diff
@@ -1,1953 +1,1963 @@
+      "release-type": "python"
+    },
+    "packages/google-geo-type": {
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "component": "google-geo-type",
+      "extra-files": [
+        "google/geo/type/gapic_version.py"
+      ],
+      "release-type": "python"
+    },
+    "packages/google-maps-addressvalidation": {
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "component": "google-maps-addressvalidation",
+      "extra-files": [
+        "google/maps/addressvalidation/gapic_version.py",
+        "google/maps/addressvalidation_v1/gapic_version.py",
+        {
+          "jsonpath": "$.clientLibrary.version",
+          "path": "samples/generated_samples/snippet_metadata_google.maps.addressvalidation.v1.json",
+          "type": "json"
+        }
+      ],
+      "release-type": "python"
+    },
+    "packages/google-maps-mapsplatformdatasets": {
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "component": "google-maps-mapsplatformdatasets",
+      "extra-files": [
+        "google/maps/mapsplatformdatasets/gapic_version.py",
+        "google/maps/mapsplatformdatasets_v1/gapic_version.py",
+        "google/maps/mapsplatformdatasets_v1alpha/gapic_version.py",
+        {
+          "jsonpath": "$.clientLibrary.version",
+          "path": "samples/generated_samples/snippet_metadata_google.maps.mapsplatformdatasets.v1.json",
+          "type": "json"
+        },
+        {
+          "jsonpath": "$.clientLibrary.version",
+          "path": "samples/generated_samples/snippet_metadata_google.maps.mapsplatformdatasets.v1alpha.json",
+          "type": "json"
+        }
+      ],
+      "release-type": "python"
+    },
+    "packages/google-maps-places": {
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "component": "google-maps-places",
+      "extra-files": [
+        "google/maps/places/gapic_version.py",
+        "google/maps/places_v1/gapic_version.py",
+        {
+          "jsonpath": "$.clientLibrary.version",
+          "path": "samples/generated_samples/snippet_metadata_google.maps.places.v1.json",
+          "type": "json"
+        }
+      ],
+      "release-type": "python"
+    },
+    "packages/google-maps-routing": {
+      "bump-minor-pre-major": true,
+      "bump-patch-for-minor-pre-major": true,
+      "component": "google-maps-routing",
+      "extra-files": [
+        "google/maps/routing/gapic_version.py",
+        "google/maps/routing_v2/gapic_version.py",
+        {
+          "jsonpath": "$.clientLibrary.version",
+          "path": "samples/generated_samples/snippet_metadata_google.maps.routing.v2.json",
+          "type": "json"
+        }
+      ],
+      "release-type": "python"
+    },
+    "packages/google-cloud-commerce-consumer-procurement": {
+      "extra-files": [
+        "google/cloud/commerce_consumer_procurement_v1/gapic_version.py",
```

Comment:
The release-please-config does not include an entry for v1alpha1. I say we defer that to a trailing PR.